lib/responders.rb

```diff
@@ -1,11 +1,5 @@
-require 'action_controller'
```

Comment:
Actually moving `autoload` declarations into the lazy load hook is not necessary.
Removing `require 'action_controller'` is the key point of this patch.

Should I revert `autoload` changes?
In my opinion, it is more clear if all codes which extend ActionController are located in the lazy load hook.